examples/decoration-example/package.json

```diff
@@ -8,7 +8,11 @@
     "build": "tsc",
     "watch": "tsc -w",
     "test": "jest",
-    "cdk": "cdk"
+    "cdk": "cdk",
+    "synth": "cdk synth",
```

Comment:
as in the previous comment I think these useful cdk-dia developer changes could be in the main package.json or somewhere else as they are not specific to the decorator example

Reply:
indeed. I added them here as I needed them while developing this example. what would be a good place do you think?

Reply:
maybe we can put it in a `contributiong.md`, like:
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

We can also link to it from the main Readme.

What do you think?